staging/src/k8s.io/kube-scheduler/config/v1alpha1/types.go

```diff
@@ -127,3 +136,64 @@ type KubeSchedulerLeaderElectionConfiguration struct {
 	// LockObjectName defines the lock object name
 	LockObjectName string `json:"lockObjectName"`
 }
+
+// Plugins include multiple extension points. When specified, the list of plugins for
+// a particular extension point are the only ones enabled. If an extension point is
+// omitted from the config, then the default set of plugins is used for that extension point.
+// If no plugin should be invoked at an extension point, an empty list must be provided.
```

Comment:
we've found the difference between `nil` and `[]` or `{}` to be extremely subtle (and in some cases, not even preserved by some serialization mechanisms - xref https://github.com/kubernetes/kubernetes/issues/44593 and https://github.com/kubernetes-sigs/yaml/pull/14#discussion_r271939039)

if you can represent an extension point is completely disabled more explicitly, I'd recommend doing so